zipkin-server/pom.xml

```diff
@@ -102,6 +103,36 @@
       </exclusions>
     </dependency>
 
+    <!-- Override log4j 2 version to avoid CVE-2021-44228 -->
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-api</artifactId>
+      <version>${log4j2.version}</version>
+      <type>jar</type>
```

Comment:
nitpick; type=jar and scope=compile are defaults and generally no need to provide them.

Reply:
@llinder would you be able to change this?